test/jaxp/javax/xml/jaxp/unittest/transform/Bug8169112.xsl

```diff
@@ -61,8 +61,6 @@
           .landscapeArea { page: land; width: 1010px; page-break-before: always; }
 
           @page { margin: 0.1in; }
-
-          }
```

Comment:
Does this affect the behavior of the test this is part of?

Reply:
Reverted.